Milestone 1 & 2 & 3

denisdiaconu opened the pull request:
finished the navbar and the footer of the page for phone, desktop, tablet size.

denisdiaconu committed:
completed nav bar and footer of the page for phone desktop  and tablet

denisdiaconu committed:
Update README.md

denisdiaconu committed:
Update index.html

denisdiaconu committed:
fixed head

denisdiaconu committed:
Update index.html

denisdiaconu committed:
Update index.html

denisdiaconu committed:
Update index.html

denisdiaconu committed:
Update index.html

denisdiaconu committed:
fix

denisdiaconu committed:
fix

denisdiaconu committed:
Update index.html

denisdiaconu committed:
Update index.html

denisdiaconu committed:
Update index.html

denisdiaconu committed:
fixed linter

denisdiaconu committed:
fixed footer

denisdiaconu committed:
fixed title

SpaYco commented on style.css:
```diff
@@ -0,0 +1,285 @@
+}
+
+.nav_bar_mid > li {
+  display: flex;
```
- [x] Use Bootstrap for Display Flex.

SpaYco commented on style.css:
```diff
@@ -0,0 +1,285 @@
+.top_nav {
+  background-color: #f72210;
+  height: 95px;
+  align-items: flex-end;
```
- [x] you can use `align-items-end` as a class.

SpaYco commented on style.css:
```diff
@@ -0,0 +1,285 @@
+#phone_navbar {
+  background-color: #f72210;
+  width: 100%;
+  display: flex;
```
- [x]  Use Bootstrap for Display Flex.

SpaYco commented on style.css:
```diff
@@ -0,0 +1,285 @@
+}
+
+#phone_cont {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```
- [x]  Use Bootstrap for Display Flex.
- [x]  Use Bootstrap for aligning Items.
- [x] Use Bootstrap for justifying content.

SpaYco commented on style.css:
```diff
@@ -0,0 +1,285 @@
+}
+
+#phone_cont > div {
+  display: flex;
+  align-items: center;
```
- [x]  Use Bootstrap for Display Flex.
- [x]  Use Bootstrap for aligning Items.

SpaYco commented on style.css:
```diff
@@ -0,0 +1,285 @@
+}
+
+#nav_hide {
+  display: none;
```
- [x] use `d-none` for `display: none;`

SpaYco commented on style.css:
```diff
@@ -0,0 +1,285 @@
+  }
+
+  #user {
+    display: none;
```
- [x] use `d-none` for `display: none;`

SpaYco commented on style.css:
```diff
@@ -0,0 +1,285 @@
+
+@media (min-width: 768px) {
+  .nav_top2 {
+    justify-content: flex-end;
```
- [x] you can use `align-items-end` as a class.

SpaYco reviewed (approved):
# Milestone Approved ✔️ 

Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone.

- [x] on the footer, the copyright text should be exactly next to the logo, without the extra space  ([Link to page](https://www.newsweek.com/))
- [x] place add the gray area on the sides of the page ([Link to page](https://www.newsweek.com/))
- [x] the text of the nav links are going above the red background, that shouldn't happen.



Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

denisdiaconu committed:
finished the layout of the page

denisdiaconu committed:
finished the 3 col and the style in css

denisdiaconu committed:
fixed part of text in col 1

denisdiaconu committed:
fixed text

denisdiaconu commented:
Added col 1, col 2, col 3 and their layout

SpaYco commented on style.css:
```diff
@@ -0,0 +1,454 @@
+
+.search_part {
+  gap: 10px;
+  align-items: center;
```
- [x] use `align-items-center` Bootstrap class for this.

SpaYco commented on style.css:
```diff
@@ -0,0 +1,454 @@
+}
+
+.col3_text {
+  font-weight: bold;
```
- [ ] Use `font-weight-bold` Bootstrap class for this

SpaYco commented on style.css:
```diff
@@ -0,0 +1,454 @@
+
+.col3_title {
+  color: black;
+  font-weight: bold;
```
- [ ] Use `font-weight-bold` Bootstrap class for this

SpaYco commented on style.css:
```diff
@@ -0,0 +1,454 @@
+
+.col2_title {
+  color: #818181;
+  font-weight: bold;
```
- [ ] Use `font-weight-bold` Bootstrap class for this

SpaYco commented on style.css:
```diff
@@ -0,0 +1,454 @@
+}
+
+#hide {
+  display: none;
```
- [x] use `d-none` Bootstrap class for this, also, don't mark these as an ID because you might use it for multiple elements.

SpaYco commented on style.css:
```diff
@@ -0,0 +1,454 @@
+
+.border_top {
+  border-top: 1px solid #d9d9d9;
+  display: flex;
```
- [x] use `d-flex` for this.

SpaYco commented on style.css:
```diff
@@ -0,0 +1,454 @@
+
+  #hide {
+    display: inline;
+    font-weight: bold;
```
- [x] Use `font-weight-bold` Bootstrap class for this

SpaYco commented on style.css:
```diff
@@ -0,0 +1,454 @@
+  }
+
+  #hide {
+    display: inline;
```
- [x] use `d-inline` Bootstrap class for this.

SpaYco reviewed (approved):
# Milestone Approved ✔️ 

Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone.

- [x]  on the third column, I don't see the **Free signup** section
- [x] The content order and disposition changes to the correct place in tablet and mobile
- [x] Use bootstrap classes instead of using your custom CSS file as much as possible, I mentioned some below 👇, but double-check if I missed some.


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

denisdiaconu committed:
fixed form

denisdiaconu committed:
finished layout of the page

denisdiaconu committed:
fixing errors

denisdiaconu committed:
fixed errors

denisdiaconu committed:
fixed

denisdiaconu committed:
fixing magazin layout

denisdiaconu committed:
fixed magazin

denisdiaconu commented:
> # Milestone Approved heavy_check_mark
> 
> Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone.
> 
>     * [ ]   on the third column, I don't see the **Free signup** section
> 
>     * [ ]  The content order and disposition changes to the correct place in tablet and mobile
> 
>     * [ ]  Use bootstrap classes instead of using your custom CSS file as much as possible, I mentioned some below point_down, but double-check if I missed some.
> 
> 
> ### please don't forget to merge to master, good luck with the next projects!
> 
> Best,
> Aziz Mejri
> _contact me on [twitter](https://twitter.com/iSpaYco) or slack (@Spayco) if you need any help_

Thank you for the review!
I will fix them until tomorrow.

denisdiaconu commented:
Added the last part of the page.

    US
    World
    Business
    Tech & Science
    Culture
    Sports
    Expert forum

denisdiaconu committed:
fixed nav bar on tablet

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                <img src="images/logo.png" alt="logo">
+            </div>
+            <div>
+                <button type="button" class="btn login" style="color: white; font-weight: bold;">Login</button>
+                <button type="button" class="btn btn-warning button" style="border-radius: 23px; font-weight: bold; color: black; height: 30px; background-color: #ffe400;">SUBSCRIBE ></button>
```
- [x] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+        </div>
+        <div >
+            <ul class="justify-content-start m-0 list-unstyled d-flex nav_bar_mid ">
+                <li style="border: none;"><a href="#">U.S.</a></li>
```
- [x] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                <li><a href="#">Experts</a></li>
+                <li><a href="#">Vantage</a></li>
+                <li class="d-flex search_cont" style="border: none;"><div class="d-flex search_part">
+                    <p style="color: #b7b7b7; margin: auto;">Search</p>
```
- [x] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                    <p style="color: #b7b7b7; margin: auto;">Search</p>
+                    <i class="fas fa-search"></i>
+                </div></li>
+                <li id="border_red" style="height: 100%;"></li>
```
- [x] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+        </div>
+        <div class="row d-flex last_nav">
+            <ul class="col-lg-10 d-flex list-unstyled justify-content-start nav_bar_bot">
+                <li style="font-weight: bold;"><span>TREDING</span></li>
```
- [x] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+            <div>
+                <h3 id="hide">LOGIN</h3>
+                <i id="user" class="fas fa-user"></i>
+                <button class="btn btn-sm" type="button" style="font-size: 12px; height: 25px; border-radius: 24px; background-color: #ffe400;"><strong>SUBSCRIBE ></strong></button>
```
- [x] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                  </div>
+                  <div class="part1">
+                        <h4><a href="#">MORE STORIES</a></h4>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                    </div>
+                    <div class="part2">
+                        <h4><a href="#">LATEST NEWS</a></h4>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                            </div>
+                        </div>
+                        <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                            <p>Featuring a unique history, uninhibited wildness and at times otherworldly natural spectacles, Hokkaido offers an experience that is completely different from the rest of Japan.</p>
+                        </div>
+                    </div>
+                    <div style="background-color: #f4f5f6;" class="px-3 py-4 border rounded first_form">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                    </div>
+                    <div style="background-color: #f4f5f6;" class="px-3 py-4 border rounded first_form">
+                        <div class="d-flex text-left align-items-center">
+                            <a href="#" class="h4" style="background-color: #f72210; padding: 5px 9px 5px 9px; font-weight: bold; color: white;">N</a>
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                        </div>
+                        <form class="row m-auto">
+                            <input placeholder="Email Address" type="Email" class="col-12">
+                            <button type="button" style="background-color: #f61602;" class="col-12 rounded bg-red px-6 py-1 mt-3 ">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                            <p>Featuring a unique history, uninhibited wildness and at times otherworldly natural spectacles, Hokkaido offers an experience that is completely different from the rest of Japan.</p>
+                        </div>
+                    </div>
+                    <section class="bg-white col-12 magazin_cont" style="padding-top: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                            <div class="d-flex test image_mag">
+                                <h5>JANUARY 22 ISSUE</h5>
+                            </div>
+                            <div class="d-flex justify-content-center" style="background-color: #f4f5f6;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                            <h3 style="background-color: #f61602; font-size: 20px; font-weight: bold; color: white; padding: 5px 8px 5px 8px;">N</h3>
+                            <h3 style="padding: 5px; font-size: 20px; font-weight: bold;">EDITOR'S PICKS</h3>
+                        </div>
+                        <section class="bg-white border-bottom col-12 editor_cont" style="padding-top: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                </ul>
+                            </div>
+                        </section>
+                        <section style="padding-top: 40px;" class="row bg-white col-12">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                    <div>
+                                        <h4 class="font-weight-bold h3">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+                                        <p>All Access + Weekly Delivery.Choose A Membership That's Perfect for You!</p>
+                                        <button class="btn font-weight-bold btn-danger" style="background-color: red;">SUBSCRIBE</button>
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                    <h5 class="title_com">NEWSLETTER</h5>
+                                </div>
+                                <div style="height: 100%;" class="">
+                                    <div class="d-flex col-12" style="height: 90%;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                      </div>
+                      <div class="part1">
+                          <h4><a href="#">MORE STORIES</a></h4>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                      </div>
+                      <div class="part2">
+                          <h4><a href="#">LATEST NEWS</a></h4>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                                  <p>The incoming president has not said what he plans to do with the tens of thousands of asylum-seekers living in limbo in crowded shelters and makeshift camps along the border.</p>
+                              </div>
+                          </div>
+                          <div class="d-flex" style="column-gap: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                          <p>Featuring a unique history, uninhibited wildness and at times otherworldly natural spectacles, Hokkaido offers an experience that is completely different from the rest of Japan.</p>
+                      </div>
+                  </div>
+                  <div style="background-color: #f4f5f6;" class="px-3 py-4 border rounded first_form">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                  </div>
+                  <div style="background-color: #f4f5f6;" class="px-3 py-4 border rounded first_form">
+                      <div class="d-flex text-left align-items-center">
+                          <a href="#" class="h4" style="background-color: #f72210; padding: 5px 9px 5px 9px; font-weight: bold; color: white;">N</a>
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                      </div>
+                      <form class="row m-auto">
+                          <input placeholder="Email Address" type="Email" class="col-12">
+                          <button type="button" style="background-color: #f61602;" class="col-12 rounded bg-red px-6 py-1 mt-3 ">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+            </div>
+            <div class="d-flex border-bottom bg-white justify-content-center" style="padding: 30px 0 10px 0;">
+              <h3 style="background-color: #f61602; font-size: 20px; font-weight: bold; color: white; padding: 5px 8px 5px 8px;">N</h3>
+              <h3 style="padding: 5px; font-size: 20px; font-weight: bold;">IN THE MAGAZINE</h3>
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+              <h3 style="background-color: #f61602; font-size: 20px; font-weight: bold; color: white; padding: 5px 8px 5px 8px;">N</h3>
+              <h3 style="padding: 5px; font-size: 20px; font-weight: bold;">IN THE MAGAZINE</h3>
+          </div>
+            <section class="bg-white magazin_cont" style="padding-top: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                  <div class="d-flex test image_mag">
+                      <h5>JANUARY 22 ISSUE</h5>
+                  </div>
+                  <div class="d-flex justify-content-center" style="background-color: #f4f5f6;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+            </section>
+              <div class="d-flex border-bottom bg-white justify-content-center" style="padding: 30px 0 10px 0;">
+                  <h3 style="background-color: #f61602; font-size: 20px; font-weight: bold; color: white; padding: 5px 8px 5px 8px;">N</h3>
+                  <h3 style="padding: 5px; font-size: 20px; font-weight: bold;">EDITOR'S PICKS</h3>
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                  <h3 style="background-color: #f61602; font-size: 20px; font-weight: bold; color: white; padding: 5px 8px 5px 8px;">N</h3>
+                  <h3 style="padding: 5px; font-size: 20px; font-weight: bold;">EDITOR'S PICKS</h3>
+              </div>
+              <section class="bg-white border-bottom col-12 editor_cont" style="padding-top: 20px;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                      </ul>
+                  </div>
+              </section>
+              <section style="padding-top: 40px;" class="row bg-white col-12">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                          <div>
+                              <h4 class="font-weight-bold h3">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+                              <p>All Access + Weekly Delivery.Choose A Membership That's Perfect for You!</p>
+                              <button class="btn font-weight-bold btn-danger" style="background-color: red;">SUBSCRIBE</button>
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+                          <h5 class="title_com">NEWSLETTER</h5>
+                      </div>
+                      <div style="height: 100%;" class="">
+                          <div class="d-flex col-12" style="height: 90%;">
```
- [ ] Use bootstrap classes when possible or put all styles within your own classes in the CSS file

euqueme commented on index.html:
```diff
@@ -0,0 +1,1686 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <title>Title</title>
```
- [x] Use a meaningful name like "NEWSWEEK CLONE" instead of using Title

euqueme reviewed (changes requested):
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)
### Hello @denisdiaconu :balloon: 

### Thank you for your submission :handshake: your project looks very similar to the original and a few changes need to be made at this point, claps for the good work done so far :clap: :clap: :clap: let's work on the following changes :construction_worker: 

### General

- [x] Readme file has some spelling mistakes, and broken screenshots let's fix them
- [x] Use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)
> each picture with its heading should be surrounded by an article tag also make use of the aside tag for the sections below the editor's picks.
- [x] NOT make [common mistakes](https://speckyboy.com/good-bad-css-practices) in your CSS styles
> I found inline styling in several lines of your HTML file please, use only bootstrap classes for styling, and only if this is not possible create your own classes for styles but all the styles should be put in the CSS file.

### Implementation requirements

* [x] Use Bootstrap CSS classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options))

> The previous reviewer put comments directly in the CSS file, but I'll also put them here you should use bootstrap classes and the CSS file should be as small as possible so let's use bootstrap classes for the following selectors:
- margin: (0, auto)
- padding: (0, auto)
- list-style: none
- width: auto
- height: auto
- text-decoration: none;
- align-items: (all that applies)
- color: (black, white, red)
- display: (all that applies)
- font-weight: (all that applies)
- flex-direction: (all that applies)
- justify-content: (all that applies)
- position: (all that applies)
- font-size: (al that applies)
- align-content: (all that applies)

### All Milestone requirements

#### Mobile Screens
##### Navbar
- [x] The login, SUBSCRIBE button, and hamburger menu needs some margin between each other **also do it for tablet screens**
- Clone
![image](https://user-images.githubusercontent.com/52209067/106059676-a8f7e780-60b8-11eb-8d91-82bf4aa9ccab.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106059708-b4e3a980-60b8-11eb-9efd-c9f43364376f.png)
- [x] The Navbar is overlapping or being overlapped by some sections make sure it has the higher z-index
- Clone
![image](https://user-images.githubusercontent.com/52209067/106068232-e020c580-60c5-11eb-8f1a-0ba1a3c0cf18.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106068278-f62e8600-60c5-11eb-83ae-02a10194a2cb.png)
##### TOP STORY to MY TURN
- [x] all the articles should have more margin on the left
- Clone
![image](https://user-images.githubusercontent.com/52209067/106066524-984c6f00-60c2-11eb-99d0-78ee347e19a2.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106066592-b4501080-60c2-11eb-8368-00d25029b9c8.png)
##### Sign up form
- [x] should be smaller in width 
- Clone
![image](https://user-images.githubusercontent.com/52209067/106068633-78b74580-60c6-11eb-9782-56ca369db67c.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106068672-88cf2500-60c6-11eb-9ed7-78fc60ef6eda.png)
##### IN THE MAGAZINE
- [x] The Section heading is not showing and the Issue magazine should be larger in height
- Clone
![image](https://user-images.githubusercontent.com/52209067/106068943-19a60080-60c7-11eb-92b1-1addcb7ca979.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106069049-5e319c00-60c7-11eb-8ac4-d9e7a057cd6d.png)
##### Main news sections (U.S., World, etc.)
- [x] The images of the small articles should take 60% of the screen width and the text should have a margin on the left to separate it from the image **The margin applies for all screen sizes**
- Clone
![image](https://user-images.githubusercontent.com/52209067/106070023-4ce98f00-60c9-11eb-98be-95edc59b1da1.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106070063-5c68d800-60c9-11eb-86d6-e61d0d58740d.png)

##### SUBSCRIBE
- [x] The images, text and subscribe button should be centered and the bold text should be on top of the images
- Clone
![image](https://user-images.githubusercontent.com/52209067/106070294-ced9b800-60c9-11eb-8c24-c83fb46839fd.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106070333-e31db500-60c9-11eb-8ef9-210903ab4d0a.png)

##### NEWSLEETTER
- [x] The image should be on top and take the whole section width and half of the section height, the text should be centered, and the email input field next to the FREE SIGN UP button
- Clone
![image](https://user-images.githubusercontent.com/52209067/106072816-7eb12480-60ce-11eb-86e5-af94796af95f.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106072842-8a9ce680-60ce-11eb-8fd0-dacf10a73207.png)

#### Tablet Screens
##### Column 1 and 2
- [x] The headings for these columns aren't showing, or are being covered by the navbar so let's give some margin on top to correct this on this screens size
- Clone
![image](https://user-images.githubusercontent.com/52209067/106060259-6551ad80-60b9-11eb-9eb8-0c23e034d045.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106060310-77cbe700-60b9-11eb-9699-915ac5d17728.png)
##### Culture & Travel and More Stories
these sections should have more separation between each other the text of the articles its almost on top of the Section headings
- Clone
![image](https://user-images.githubusercontent.com/52209067/106073410-a359cc00-60cf-11eb-8fcf-c770c607c8da.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106073450-b53b6f00-60cf-11eb-81d7-b0e955de44f5.png)
##### Sponsored Insight / Rankins and the FREE SIGN UP form
- [x] should be smaller and  center on the page
Clone
![image](https://user-images.githubusercontent.com/52209067/106073670-21b66e00-60d0-11eb-8b62-11a5ddbfe0ea.png)
Original
![image](https://user-images.githubusercontent.com/52209067/106073721-372b9800-60d0-11eb-9fff-58b0572f2837.png)
##### Editor's Picks
the images should be larger in height
Clone
![image](https://user-images.githubusercontent.com/52209067/106076056-70660700-60d4-11eb-89dc-d06ba2afed22.png)
Original
![image](https://user-images.githubusercontent.com/52209067/106076088-7d82f600-60d4-11eb-935e-531cbf7cf428.png)
##### SUBSCRIBE 
- [x] The red heading should have a smaller font size, the text and subscribe button should have a margin on the left to separate them from the overlapping images
- Clone
![image](https://user-images.githubusercontent.com/52209067/106076731-c4bdb680-60d5-11eb-8288-1ec67cfde8b0.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106076765-d43cff80-60d5-11eb-8725-6c1b8482cec7.png)
##### NEWSLETTER

- [x] The red heading should have a smaller font size, the image should be along with the section heading, there's a gray border on the right part containing the text, the text is centered in the right part and the Email address input is next to the FREE SIGN UP button
Clone
![image](https://user-images.githubusercontent.com/52209067/106077296-e9fef480-60d6-11eb-97eb-a3137b98e8cd.png)
Original
![image](https://user-images.githubusercontent.com/52209067/106077333-f8e5a700-60d6-11eb-9f32-9f27530e29ed.png)

#### Desktop Screens
##### Navbar
- [x] The date and SUBSCRIBE buttons are on the edge of the screen let's add some margin
![image](https://user-images.githubusercontent.com/52209067/106059003-a648c280-60b7-11eb-923d-84a6c99f99e2.png)
- [x] The Navbar should be centered in the page like the following sections so let's also add some margin 
- Clone
![image](https://user-images.githubusercontent.com/52209067/106059316-27a05500-60b8-11eb-98de-0bc23a7c2ae1.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106059379-4272c980-60b8-11eb-8a01-0142d13dfde9.png)
##### IN THE MAGAZINE to Footer
- [x] From the IN THE MAGAZINE to the Footer should be centered in the page having the same margin on the left and right like the previous sections
Clone
![image](https://user-images.githubusercontent.com/52209067/106078054-58908200-60d8-11eb-892b-e9e45612344b.png)
Original
![image](https://user-images.githubusercontent.com/52209067/106078122-78c04100-60d8-11eb-807c-44a5a7901fb5.png)

#### All Screens
##### MORE STORIES and LATEST NEWS
- [x] The articles should have more margin between the image and the text
Clone
![image](https://user-images.githubusercontent.com/52209067/106060829-2bcd7200-60ba-11eb-8dc5-aab0216fd755.png)
Original
![image](https://user-images.githubusercontent.com/52209067/106065478-fe37f700-60c0-11eb-8f85-9ea9f276aaa2.png)
##### SPONSORED INSIGHT/ RANKINS
- [x] The images for these sections aren't showing 
- Clone
![image](https://user-images.githubusercontent.com/52209067/106067364-2a08ac00-60c4-11eb-8f3b-eed7cbbd30e8.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106067427-40af0300-60c4-11eb-99a4-f9a65dfd8f14.png)
##### IN THE MAGAZINE
- [x] The font size for the articles of this section should be smaller
- Clone
![image](https://user-images.githubusercontent.com/52209067/106069233-be284280-60c7-11eb-8e84-59a76a55c975.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106069263-ced8b880-60c7-11eb-923e-933ac48d6a36.png)
##### The main sections (U.S., World, etc.)
- [x] The fonts should be smaller
- Clone
![image](https://user-images.githubusercontent.com/52209067/106069780-c3d25800-60c8-11eb-99be-56e562f22f6a.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106069813-d351a100-60c8-11eb-8aa6-6729d976e24d.png)
##### Footer
- [x] The footer should have more margin on top to separate it from the NEWSLETTER, the social media icons should have more margin left or right to separate them from each other, the editions should be aligned with the NEWSWEEK logo and social media icons (**Also on Tablet screens**)
- Clone
![image](https://user-images.githubusercontent.com/52209067/106073087-0008b700-60cf-11eb-97cc-e904902a7976.png)
- Original
![image](https://user-images.githubusercontent.com/52209067/106073120-1282f080-60cf-11eb-83cb-a298b8f52266.png)
- [x] The NEWSWEEK logo should be a link
##### All images
- [x] should be links

### That's it,  :raised_hands: 
### Please :pray:  do the changes :technologist: and submit back for a code review when you are done :memo: 
Any question? ask me on slack **@ Maru** :woman_technologist: 
### Happy coding! :rocket: 
PROJECT STATUS: REQUESTED CHANGES :traffic_light:

denisdiaconu committed:
fix:

denisdiaconu committed:
fixed README

denisdiaconu committed:
fixed milestone 1 and 2

denisdiaconu committed:
Merge branch 'test_project' of https://github.com/denisdiaconu/Newsweek-clone into test_project

denisdiaconu committed:
made all the changes

denisdiaconu committed:
fixed all the content of the rev

denisdiaconu committed:
fixed margin navbar

denisdiaconu commented:
Thanks for the review!

Fix:

- The copyright text in the footer is next to the logo.
- Added gray area on the sides of the page
- The text of the nav links isn't above the red background now.
- Added the **Free signup** section.
- The content of the page changes to the correct place now for tablet, mobile, and desktop.
- Fixed spelling mistakes inside README and repaired the screenshot.
- Changed div with semantic HTML.
- Repaired inline styling.
- Used Bootstrap CSS on every single element.
- Added margin between the items of the phone & tablet navbar.
- Added tighter z-index so navbar is overlapping.
- Added margin to the left to all the articles.
-  The signup form is smaller in width now.
- The section heading of the magazine section is showing now.
- Fixed the size of the images on the small articles.
- The images, text and subscribe button inside the subscribe section are centered now.
- The images, text, and **Free sign up** button of the newsletter section are centered now.
- The headings of the columns are showing now.
- **My turn and Culture & Travel** have more space between each other now.
- The sponsored part of the page is now smaller and centered
- The images of the Editor's pick section are now higher.
- The read heading of the subscribe and Newsletter section is now smaller.
- Added some margin to the date and subscribe buttons.
- Added margin between articles.
- The images of the sponsored and ranking section are showing now.
- Fixed the font size of some articles.
- Added margin to the footer.
- Made all the images into links.

Thank you for the review.

SpaYco reviewed (changes requested):
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] your subscribe section is breaking on mobile.
- [x] fix the previous notes about CSS properties that can be Bootstrap classes.

![image](https://user-images.githubusercontent.com/11758151/106173443-36f4d080-6194-11eb-9fb7-e215030a80c5.png)


### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

denisdiaconu committed:
added bootstrap classes

denisdiaconu committed:
added more bootstrap

denisdiaconu commented:
I talked with the last code reviewer and he told me that the subscribe section is ok.
Is normal for the text to be inside the images.
I did my best and I changed everything that could be changed in bootstrap.
Only a few remain but they are in big groups.
Thank you for the review!

voscarmv reviewed (approved):
# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️
